UnitsNet/UnitDefinitions/MassFlow.json

```diff
@@ -19,8 +19,8 @@
     {
       "SingularName": "KilogramPerHour",
       "PluralName": "KilogramsPerHour",
-      "FromUnitToBaseFunc": "x/3.6",
-      "FromBaseToUnitFunc": "x*3.6",
+      "FromUnitToBaseFunc": "x/0.277778",
```

Comment:
I don't think this change is right. Google says `3.6`.

https://www.google.no/search?q=kilograms+Per+Hour+In+1+Gram+Per+Second